android/src/androidTest/java/com/getcapacitor/android/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.getcapacitor.android
+
+import android.content.Context
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see [Testing documentation](http://d.android.com/tools/testing)
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    @Throws(Exception::class)
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext: Context = InstrumentationRegistry.getInstrumentation().targetContext
+
+        assertEquals("com.getcapacitor.android", appContext.packageName)
+    }
+}
```

Comment:
Since we're not testing now, let's completely get rid of this test and possibly the whole suite. We can always bring it back if we need.